views.py

```diff
@@ -396,7 +401,7 @@ class LessonView(HTTPMethodView):
             lesson = await Lesson.get_by_id(lid)
             return json(await lesson.to_dict())
         else:
-            lessons = await Lesson.get_all()
+            lessons = await Lesson.get_all(suffix="ORDER BY lesson_no")
```

Comment:
+1

Reply:
Not sure whether this works as expected or not, though; I'll check that in a sec

Reply:
well, actually it does :D